pkg/detectors/fibery/fibery.go

```diff
@@ -0,0 +1,88 @@
+package fibery
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	//Make sure that your group is surrounded in boundry characters such as below to reduce false positives
+	keyPat    = regexp.MustCompile(detectors.PrefixRegex([]string{"fibery"}) + `\b([0-9a-z.]{44})\b`)
+	domainPat = regexp.MustCompile(detectors.PrefixRegex([]string{"fibery", "domain"}) + `\b([0-9a-z]{5,20})\b`)
```

Comment:
suggestion we can also make this a little more robust. After some testing it looks like they allow alphanumerics as part of the domain name and also allow a minimum of 2 characters and a max of around 40. So maybe lets go with `\b([0-9A-Za-z]{2,40})\b` ?